cron/client.py

```
@@ -220,82 +228,109 @@ def reboot():
+                            raise RuntimeError(f"Temperature could not be stabilized in time. Was {exc.temperature} but should be {config['machine']['base_temperature_value']}. Please check logs ...") from exc
+                        if exc.direction == 'hot':
+                            print(f"Machine is still too hot: {exc.temperature}°. Sleeping for 1 minute")
+                            set_status('cooldown', data=str(exc), run_id=job._run_id)
+                            temperature_cooldown_time += 60
+                            temperature_errors += 1
+                            time.sleep(60)
+                        else:
+                            print(f"Machine is too cool: {exc.temperature}°. Warming up and retrying")
+                            set_status('warmup', data=str(exc), run_id=job._run_id)
+                            temperature_errors += 1
+                            subprocess.check_output('for i in $(seq $(nproc)); do yes > /dev/null & done', shell=True, encoding='UTF-8', errors='replace')
```

Ruff (0.16.1)

[error] `subprocess` call with `shell=True` seems safe, but may be changed in the future; consider rewriting without `shell`

(S602)